src/test/java/com/google/api/generator/gapic/composer/grpcrest/goldens/EchoClientHttpJsonTest.golden

```diff
@@ -805,4 +814,16 @@ public class EchoClientHttpJsonTest {
       // Expected exception.
     }
   }
+
+  @Test
```

Comment:
Is it possible to not generate this whole test class if this case happens? I know it would be a rare case, so if it takes a lot of effort, I think this is fine.

Reply:
It is much-much simpler to generate empty tests, but I believe it is a better approach to have empty tests instead of not having a class (eve if it was harder to do)